scripts/build/esbuild-plugins/throw-warnings.mjs

```diff
@@ -60,13 +60,14 @@ export default function esbuildPluginThrowWarnings({ allowDynamicRequire }) {
           if (
             (warning.location.file ===
               "node_modules/flow-parser/flow_parser.js" ||
-              warning.location.file.startsWith("dist/_parser-flow.js")) &&
+              warning.location.file === "dist/_parser-flow.js") &&
```

Comment:
Note to myself, can't use this filename, if we build `.js` and `.mjs` in parallel will cause problem. Can't use `outfile` either because we support `--save-as`. Will figure how to fix this later.